netkan/netkan/cli/common.py

```diff
@@ -91,23 +90,31 @@ def ssh_key(self, value):
         init_ssh(value, Path(Path.home(), '.ssh'))
         self._ssh_key = value
 
+    @property
+    def ckanmeta_repo(self):
+        if not self._ckanmeta_repo:
+            self._ckanmeta_repo = CkanMetaRepo(
+                init_repo(self._ckanmeta_remote, '/tmp/CKAN-meta', self.deep_clone))
```

Comment:
We could move the Getter/Setter stuff into XKanaMetaRepo, which would clean up the need for us to repeat this in the CLI multiple times.

Reply:
Well, we still need to _create_ the `*KanRepo` objects somewhere. And we're not repeating, here; the remote and path parameters are different in each call.